src/yokozuna_rt.erl

```diff
@@ -264,6 +274,20 @@ search_expect(solr, {Host, Port}, Index, Name0, Term0, Shards, Expect)
     {ok, "200", _, R} = ibrowse:send_req(URL, [], get, [], Opts),
     verify_count_http(Expect, R).
 
+assert_search(Pid, Cluster, Index, Search, SearchExpect, Params) ->
+    F = fun(_) ->
+                lager:info("Searching ~p and asserting it exists",
+                           [SearchExpect]),
+                {ok,{search_results,[{_Index,Fields}], _Score, Found}} =
+                    riakc_pb_socket:search(Pid, Index, Search, Params),
```

Comment:
I like the idea of abstracting this out, but we should probably use the pattern used in

https://github.com/basho/yokozuna/blob/2.0/riak_test/yz_pb.erl#L233

where we also pattern match against an empty set of results.  I don't know if we will get [] results in these tests, but I did hit them in the yz_pb tests.